src/constants.ts

```diff
@@ -0,0 +1,27 @@
+export const MS_IN_SEC = 1000
+export const SEC_IN_MIN = 60
+export const MIN_IN_HOUR = 60
+export const HOUR_IN_DAY = 24
+export const DAYS_IN_WEEK = 7
+export const ADDITIONAL_UNIT = 1
+export const NO_DIFF = 0
+export const RECURRING_TASK_LIMIT = 24
+export const LOWEST_TRANSFERRABLE_AMOUNT = 1000000000
+
+export const NEUMANN_WEBSOCKET = 'wss://rpc.testnet.oak.tech'
+export const TURING_WEBSOCKET = 'wss://rpc.turing.oak.tech'
+
+export enum OakChainWebsockets {
+  NEU = 'wss://rpc.testnet.oak.tech',
```

Comment:
can you set this to `NEUMANN_WEBSOCKET`?  ie `NEU = NEUMANN_WEBSOCKET`. Better if we don't have to update the same value in two places.

Reply:
`Computed values are not permitted in an enum with string valued members.` is the error when i try haha. i will remove the other 2 consts and just use the enum